shared/menubar/index.js

```diff
@@ -30,25 +30,14 @@ export type MenubarProps = {
 
 class Menubar extends Component {
   props: MenubarProps;
-  state: {
-    loading: boolean;
```

Comment:
Not yet. That's what Max is architecting now

On Friday, March 4, 2016, Marco Munizaga <notifications@github.com> wrote:

> In shared/menubar/index.js
> <https://github.com/keybase/client/pull/2228#discussion_r55112591>:
>
> > @@ -30,25 +30,14 @@ export type MenubarProps = {
> >
> >  class Menubar extends Component {
> >    props: MenubarProps;
> > -  state: {
> > -    loading: boolean;
>
> Do we get a notification when a user adds a favorite?=
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/2228/files#r55112591>.
>